<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VlanConfig</class>
 <widget class="QDialog" name="VlanConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vlan配置</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLineEdit" name="com_num">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>48</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="first_vlan">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="com_type">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>gi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="slot_num">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>86</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>端口数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>槽位号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>vlan起始编号：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="auto_gen">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>自动生成</string>
   </property>
  </widget>
  <zorder>com_num</zorder>
  <zorder>com_type</zorder>
  <zorder>slot_num</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>cancel</zorder>
  <zorder>auto_gen</zorder>
  <zorder>first_vlan</zorder>
 </widget>
 <tabstops>
  <tabstop>com_num</tabstop>
  <tabstop>com_type</tabstop>
  <tabstop>slot_num</tabstop>
  <tabstop>first_vlan</tabstop>
  <tabstop>auto_gen</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
